NEWS.md

```diff
@@ -14,7 +14,26 @@
 ### Other breaking changes
 
 - Several functions have been rewritten to match the behavior of Python Polars.
-  - In `pl$Series()`, the first argument `x` is renamed to `values` (#933).
+  - In `pl$Series()` arguments are changed.
+    - The argument `x` is renamed `values` (#933).
+    - The argument `values` has a new default value `NULL` (#966).
+    - Using positional arguments in `pl$Series()` throws a warning, since the
+      argument positions will be changed in the future (#966).
+
+      ```r
+      # polars 0.15.1 or earlier
+      # The first argument is `x`, the second argument is `name`.
+      pl$Series(1:3, "foo")
+
+      # The code above will warn in 0.16.0
+      # Use named arguments to silence the warning.
+      pl$Series(values = 1:3, name = "foo")
+      pl$Series(name = "foo", values = 1:3)
+
+      # polars 0.17.0 or later (future version)
+      # The first argument is `name`, the second argument is `values`.
+      pl$Series("foo", 1:3)
+      ```
```

Comment:
```suggestion
      ```
      This warning can also be silenced by replacing `pl$Series(<values>, <name>)` 
      by `as_polars_series(<values>, <name>)`.
```

Reply:
We should advertise `as_polars_series()` as a way to silence this warning